Frameworks/Plugins/ZappCrashlogsMsAppCenter/Manifest/ios.json

```diff
@@ -4,16 +4,19 @@
     "class_name": "ZappCrashlogsMsAppCenter.ZappCrashlogsMsAppCenter",
     "modules": []
   },
-  "dependency_repository_url": [],
+  "dependency_name": "@applicaster/crash-logs-ms-app-center",
+  "npm_dependencies": [
+    "@applicaster/crash-logs-ms-app-center@<%= version_id %>"
+  ],
   "extra_dependencies": [
     {
-      "ZappCrashlogsMsAppCenter": ":git => 'https://github.com/applicaster/AppleApplicasterFrameworks.git', :tag => 2020.56.5.15-1-5"
+      "ZappCrashlogsMsAppCenter": ":path => './node_modules/crash-logs-ms-app-center/ZappCrashlogsMsAppCenter.podspec'"
```

Comment:
This comment is true for all plugins in this repo.
In npm package names, you can have a scope (like a directory). this is what the `@applicaster` is meant for.
if the package name has `@applicaster/<package-name>` in it, then the path in node_modules is `./node_modules/@applicaster/<package-name>`
if not, the path is `./node_modules/<package-name>`